OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.Exceptions.Retrieve.cs

```diff
@@ -54,8 +54,11 @@ public void ShouldThrowServiceExceptionOnRetrieveAllWhenExceptionOccursAndLogIt(
             // given
             var serviceException = new Exception();
 
+            var failedCourseServiceException =
```

Comment:
This change should be from another user branch. Merge latest to just show exam attachment changes.